src/iconid.rs

```diff
@@ -1,19 +1,31 @@
 //! Identification of icons and resolution of glyph ids. Assumes Google style icon font input.
+//!
+use crate::error::IconResolutionError;
 use skrifa::{
     charmap::Charmap,
     instance::LocationRef,
     raw::{
         tables::{
-            gsub::{Gsub, LigatureSubstFormat1, SingleSubst, SubstitutionSubtables},
+            gsub::{
+                Gsub, LigatureMarker, LigatureSubstFormat1, SingleSubst, SubstitutionSubtables,
+            },
             layout::ConditionSet,
         },
-        FontRef, ReadError, TableProvider, TopLevelTable,
+        FontRef, ReadError, TableProvider, TableRef, TopLevelTable,
     },
-    GlyphId,
+    GlyphId, MetadataProvider,
 };
 use smol_str::SmolStr;
+use std::{
+    collections::{
+        hash_map::Entry::{Occupied, Vacant},
+        HashMap, VecDeque,
+    },
+    ops::RangeInclusive,
+};
 
-use crate::error::IconResolutionError;
+const _ICON_CODEPOINTS: [RangeInclusive<u32>; 3] =
```

Comment:
should be _PUA_CODEPOINTS  and reference something, perhaps https://en.wikipedia.org/wiki/Private_Use_Areas